scopes/harmony/config/workspace-template.jsonc

```diff
@@ -55,9 +55,10 @@
      **/
     "*": {
       /**
-       * uncomment to apply the react environment on all components.
+       * uncomment to apply the chosen environment on all components.
        **/
       // "teambit.react/react": { }
+      // "teambit.angular/angular": { }
```

Comment:
@ocombe 
let's remove this from the template, for now, users might uncomment it

Reply:
that shouldn't be a problem, I'm merging into the angular branch, not master :)